components/MovieDetails/index.js

```diff
@@ -2,29 +2,34 @@ import Image from "next/image";
 import styled from "styled-components";
 import Link from "next/link";
 
-/*const MovieDetailsWrapper = styled.section`
+const MovieDetailsWrapper = styled.section`
   border: 1px solid black;
   background-color: var(--color-background-500);
-`;*/
+`;
 
 const MovieDetails = ({ data }) => {
   const currentMovie = data.find((movie) => movie.id === id);
 
   return (
     <>
-      <div>
+      <MovieDetailsWrapper>
         <Image
           src={currentMovie.image}
           alt="Movie Poster"
           width={150}
           height={220}
         />
+        <ul>
+          <li></li>
+          <li></li>
+          <li></li>
+        </ul>
```

Comment:
please don't display an empty list. did you mean to display the movie information here?